analytics/src/analytics/cli.py

```diff
@@ -65,33 +66,24 @@ def calculate_sprint_burndown(
     sprint_file: Annotated[str, SPRINT_FILE_ARG],
     issue_file: Annotated[str, ISSUE_FILE_ARG],
     sprint: Annotated[str, SPRINT_ARG],
-    *,  # makes the following args keyword only
     unit: Annotated[Unit, UNIT_ARG] = Unit.points.value,  # type: ignore[assignment]
+    *,  # makes the following args keyword only
     show_results: Annotated[bool, SHOW_RESULTS_ARG] = False,
     post_results: Annotated[bool, POST_RESULTS_ARG] = False,
 ) -> None:
     """Calculate the burndown for a particular sprint."""
-    # defer load of settings until this command is called
-    # this prevents an error if ANALYTICS_SLACK_BOT_TOKEN env var is unset
-    from config import settings
-
     # load the input data
     sprint_data = SprintBoard.load_from_json_files(
         sprint_file=sprint_file,
         issue_file=issue_file,
     )
     # calculate burndown
     burndown = SprintBurndown(sprint_data, sprint=sprint, unit=unit)
-    # optionally display the burndown chart in the browser
-    if show_results:
-        burndown.show_chart()
-    # optionally post the results to slack
-    if post_results:
-        slackbot = slack.SlackBot(client=WebClient(token=settings.slack_bot_token))
-        burndown.post_results_to_slack(
-            slackbot=slackbot,
-            channel_id=settings.reporting_channel_id,
-        )
+    show_and_or_post_results(
```

Comment:
🎉 ! 

I think this will be helpful when we add more options, such as "write to s3".